alfresco-remote-testrunner/build.gradle

```diff
@@ -47,7 +65,7 @@ jar {
             'Bundle-SymbolicName': "${project.group}.${project.name}",
             'Bundle-Version': "${project.version.version.normal}",
             'Alfresco-Spring-Configuration': 'eu.xenit.testing.integrationtesting',
-            'Import-Package': 'org.alfresco.*,org.springframework.*;version=\"[3.0,999)\",com.github.dynamicextensionsalfresco.*,org.osgi.*,org.slf4j.*',
+            'Import-Package': 'org.alfresco.*,org.springframework.*;version=\"[3.0,999)\",com.github.dynamicextensionsalfresco.*,org.osgi.*,org.slf4j.*,org.apache.http.*',
```

Comment:
We do not import those http packages.
We have them internally, so no need to change this.